GameDefinition/Content/CaveKind.hs

```diff
@@ -373,6 +375,9 @@ shallowRogue = rogue
                     : filter ((/= "treasure") . fst) (citemFreq rogue)
   , cdesc         = "This close to the outer level, residence is not permitted and doors are sturdier, to contain a possible micro-meteorite breach. The passage upwards is not closed off, though, because some passengers can't live without regularly looking at the void and the light of distant suns and planets through the reinforced glass of oriels, as opposed to the glass of electronic displays. Animals appear to share the fascination, or perhaps they crave the increased gravity, nearly Earth-like, unlike elsewhere on the ship."
   }
+  
+-- * On-planet caves
```

Comment:
Geeky nitpick: this is actually not a planet, but a moon. But indeed we could in the future have scenarios on planets, so the general category would be "celestial bodies".

Reply:
And the general division is that on a spaceship, even a huge one, biospheres can never be so deep, because there's not enough space. So we have rather bushes than trees, rather streams and pools than small rivers and lakes, rather cramped hydroponics than semi-naturally grown plants, etc.